README.md

```diff
@@ -2,92 +2,82 @@
 
-```
-cd tests
-pytest
-```
+### Automated Rinkeby deployment via Travis CI
 
-## Docker
+Travis CI is set up automatically deploy the contracts to the [Rinkeby testnet](https://www.rinkeby.io) after the tests run successfully.  These deployments happen on a tagged version (release or pre-release) of augur-core, pushes changes to [augur-contracts](https://github.com/AugurProject/augur-contracts), and updates augur.js to match that new version of augur-contracts.
 
-You may run augur-core using docker as follows. All docker commands map to the non-dockerized versions where make sense. All docker commands beginning with docker:run will execute the command within the docker image, all commands without run (e.g. docker:test) will first build the image and then execute docker:run:<command>.
+1. Tag augur-core as needed for a versioned deploy, e.g for a pre-release deployment: `npm version prerelease`
+2. CI will Build, Test, and Deploy the contracts
+3. augur-contracts will automatically have a new dev-channel version published to NPM (augur-contracts@dev)
+4. augur.js will automatically have a new dev-channel version published to NPM (augur.js@dev)
+5. If this is a real release (i.e. we want to declare this fit for public consumption), augur-contracts and augur.js should be published manually to NPM with their next version numbers.
+6. If this is a real release, augur-contracts and augur.js should be incremented to their NEXT release version number, and have the pre-release version -0 appended to them, and pushed to master.
 
-### Build:
+### Local deployment
 
-```
-npm run docker:build
-```
+Deployment can be run in two modes, direct or Docker. In direct mode, the assumption is that the entire system has been built (TypeScript and Solidity), and there is a working Node.js environment locally. Furthermore, one must possess an account on the deployment target (e.g. Rinkeby testnet) which has enough ETH to cover the costs of the deployment. For the purposes of deploying to the testnets, those with access to the augur private testnet keys can deploy and update the existing contracts. For reference, these keys are stored in an encrypted git repository within the Augur Project's keybase team. If you need access, please inquire in the Augur Discord.
 
-### Test:
+All deployment commands can be managed through scripts in package.json, and are accessible to `npm run` and `yarn`.
 
-```
-# With a pre-built image
-npm run docker:run:test:unit:all
+### Manual Rinkeby deployment
 
-# Build and run all unit tests and integration tests
-npm run docker:test
+Build and create compile artifacts:
 
-# Build and run just integration test
-npm run docker:test:integration
+```bash
+augur-core % npm run build
```

Comment:
Recommend removing the `augur-core % ` prefix on these commands.  It is unclear to the casual observer whether that is part of the command or just how _your_ prompt looks.  Many people have prompts that look different, and this isn't even the default prompt.

Reply:
Good call.